### ISSUE TITLE
How to install for a specific coq version

### ISSUE THREAD

**Issue by brando90** [unicoq/unicoq#69]
Hi,

I want to use this plugin for this coq (and coq-serapi) versions:
```
coq                     8.11.0        pinned to version 8.11.0
coq-serapi              8.11.0+0.11.0 pinned to version 8.11.0+0.11.0 at git+https://github.com/ejgallego/coq-serapi.git#v8.16
```

**Comment by beta-ziliani** [comment]
Oh I don't see a release for 8.11, maybe we missed it? There were a couple of fixes that landed in the release for 8.12 that are worth getting in, so if you could move to ≥8.12, that would be best.

**Comment by beta-ziliani** [comment]
No response, I'm closing this issue. Feel free to reopen.
